chandra_cmd_states/__init__.py

```diff
@@ -1,4 +1,23 @@
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
+import os
+import warnings
+
+message_warn = (
+    "\n"
+    "The chandra_cmd_states package is no longer supported and is replaced\n"
+    "by kadi.commands.states. Please update your code accordingly.\n"
+)
+message_error = (
+    "\n"
+    "To temporarily continue using this package set the environment variable\n"
+    "SKA_ALLOW_DISCONTINUED_PACKAGES=1"
```

Comment:
Picking nits, I think that's SKA_ALLOW_DISCONTINUED_PACKAGES='1'.  but I'm good!